spec/services/reject_npq_application_spec.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.shared_examples "validating an application to be rejected" do
```

Comment:
do we need the shared examples as this is only for NPQ, same for other spec?

Reply:
Hi @cwrw tks for your review. Addressed in https://github.com/DFE-Digital/early-careers-framework/pull/2674/commits/8c9fde545669f063b8a66cd2c524248a8025fdb5